clippy_lints/src/returns.rs

```diff
@@ -22,13 +24,15 @@ declare_lint! {
     "using a return statement like `return expr;` where an expression would suffice"
 }
 
-/// **What it does:** This lint checks for `let`-bindings, which are subsequently returned.
+/// **What it does:** Checks for `let`-bindings, which are subsequently returned.
 ///
-/// **Why is this bad?** It is just extraneous code. Remove it to make your code more rusty.
+/// **Why is this bad?** It is just extraneous code. Remove it to make your code
+/// more rusty.
 ///
-/// **Known problems:** Following this lint's advice may currently run afoul of Rust issue
-/// [#31439](https://github.com/rust-lang/rust/issues/31439), so if you get lifetime errors, please
-/// roll back the change until that issue is fixed.
+/// **Known problems:** Following this lint's advice may currently run afoul of
+/// Rust issue [#31439](https://github.com/rust-lang/rust/issues/31439), so if
```

Comment:
the new issue is https://github.com/rust-lang/rust/issues/33490, and it only occurs without MIR. Since the default is MIR nowadays.... remove the problem note entirely?